Registration

llD3vi1ll opened the pull request.

llD3vi1ll committed:
CreateAccountPrage

Wrote script for createAccontPage for testing.

llD3vi1ll committed:
Login, Logout, creating account

llD3vi1ll committed:
Changes in create, login, logout test

llD3vi1ll committed:
Merge branch 'prestaging' of github.com:CodeBud3/Automation-Testing into Registration

llD3vi1ll committed:
aa

llD3vi1ll committed:
LOGINPAGE & CREATE ACCOUNT

1) optimized the login page.
2) 17 Test cases in login page.
3) CREATE ACCOUNT created for registration.
4) revert to old chrome.
5) added ENUM.

llD3vi1ll committed:
Registration page and Login page

Registration page and Login page. test cases are completed and script is optimized. Ready for the review.

llD3vi1ll commented:
Registration page and Login page. Test cases are completed and script is optimized. Ready for the review.

RAKESH13595 commented on src/main/resources/config.properties:
```diff
@@ -17,4 +23,4 @@ implicitWait = 10
 #admin auth token
-token = ~
\ No newline at end of file
+token = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2N2Q5YzRjYjE2MzAzMWU5OGZhMTIwYmEiLCJyb2xlIjoiYWRtaW4iLCJpYXQiOjE3NDI2NzQyMTJ9.77BBNFOPA3bSmWGiKBzNFNSxHnFpZjGq1pwC6os8H4k
```
Please remove this @llD3vi1ll

llD3vi1ll committed:
Added forgot password page

3 New testcase added to login page and created forgot password page with 2 test case, deleted logout page.

llD3vi1ll commented:
@RAKESH13595 I have removed the token and 3 New test case added to login page and created forgot password page with 2 test case, deleted logout page.

RAKESH13595 reviewed (approved):
LGTM
